src/main.cpp

```diff
@@ -2398,11 +2398,16 @@ bool static DisconnectTip(CValidationState &state) {
     mempool.check(pcoinsTip);
     // Update chainActive and related variables.
     UpdateTip(pindexDelete->pprev);
+    // Get the current commitment tree
+    ZCIncrementalMerkleTree newTree;
+    assert(pcoinsTip->GetAnchorAt(pcoinsTip->GetBestAnchor(), newTree));
```

Comment:
Not at present, but we might want to use it for #1302 (probably just for verifying that we get the same root). I did it this way for a) symmetry, and b) in case we found other uses for this signal in future. But I can certainly just pass an empty tree if we want (which would only be an efficiency improvement by commenting out the `GetAnchorAt()` line).